changelog/0084-add-geospatial-queue.yml

```diff
@@ -0,0 +1,27 @@
+title: New Geospatial Celery Queue
+key: add-geospatial-queue
+date: 2024-11-06
+optional_per_env: no
+min_commcare_version:
+max_commcare_version: 513700958cf443a85f0480ff681979d207f22db0
+context: |
+  The Microplanning feature requires the location properties on cases to be indexed correctly so
+  that they can be used correctly in this feature. A separate queue is added to handle this indexing
+  process when the Microplanning feature is enabled for a domain. This can potentially be a long-running
+  task, and so having a separate queue ensures that other queues do not get clogged up by this indexing task.
+
+details: |
+  This creates a geospatial_queue in Celery, which will be used for indexing cases when enabling the
+  Microplanning feature.
+
+update_steps: |
+  This update should be performed before updating CommCare to a version more recent than that specified above.
+
+  First, add a queue called `geospatial_queue` to your app_processes.yml,
+  following the examples in commcare-cloud commit a94636b1d.
+
+  To apply this change, run:
```

Comment:
nit:
`Then, to apply this change`

Reply:
Can we note that a restart is needed for the change to take effect and can be done as a part of the following command or a deploy or done separately?

Reply:
Sure, I have refactored the text and have added a new paragraph at the bottom explaining that a restart is required ([0528be5](https://github.com/dimagi/commcare-cloud/pull/6435/commits/0528be517f9a1234208b0c1f8afc8e99b7cb3fb0)).